package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@sixa/wp-react-hooks",
-	"version": "1.2.1",
+	"version": "1.2.2",
```

Comment:
Given that we're adding functionality, would it be more sensible if we updated the version to `1.3.0` instead?

Reply:
Sure, I agree.